### PR TITLE
remove emoji from string transformer. bump down carbon

### PR DESCRIPTION
**Pull request by ruskiyos**

### CODE REVIEW

File: tests/unit/Transformer/EmojiStringTransformerCest.php [hunk 1]
```diff
@@ -2,13 +2,13 @@
 declare(strict_types = 1);
 
 use Diaclone\Resource\Item;
-use Diaclone\Transformer\StringTransformer;
+use Diaclone\Transformer\EmojiStringTransformer;
 
-class StringTransformerCest
+class EmojiStringTransformerCest
```
Comment (iampersistent) [inline tests/unit/Transformer/EmojiStringTransformerCest.php]: Does this leave us without a test for `StringTransformer`?

**Comment by iampersistent** [comment]
@ruskiyos thanks for this

File: tests/unit/Transformer/EmojiStringTransformerCest.php (hunk 1 above)
Comment (ruskiyos) [reply tests/unit/Transformer/EmojiStringTransformerCest.php]: yes. i don't think we ever had a string cest

File: src/Transformer/EmojiStringTransformer.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace Diaclone\Transformer;
+
+use Diaclone\Exception\MalformedInputException;
+use Diaclone\Exception\TransformException;
+use Diaclone\Resource\ResourceInterface;
+use Diaclone\Traits\EmojiTrait;
+use Exception;
+
+class EmojiStringTransformer extends AbstractTransformer
```
Comment (alexcoffee) [inline src/Transformer/EmojiStringTransformer.php]: `EmojiStringTransformer` sounds like it's made for converting emojis but in reality it serves the purpose of converting all our textual fields. It also does html entity encoding/decoding. Something tells me we might add another string pre-processing step here in the future. In the future we might think about refactoring into a generic string transformer with options.

Comment (ruskiyos) [reply src/Transformer/EmojiStringTransformer.php]: how do you anticipate passing in params?
```
$transformers = [
   'title' => StringTransformer::class . ":uuid,emoji",
]
```

**Pull request merged**